aiokafka/consumer/group_coordinator.py

```diff
@@ -421,7 +439,12 @@ async def _perform_assignment(
                   " with subscriptions %s", self.group_id, assignor.name,
                   member_metadata)
 
-        assignments = assignor.assign(self._cluster, member_metadata)
+        if isinstance(assignor, AbstractStaticPartitionAssignor):
```

Comment:
Maybe we could put a warning if `group_instance_id` is not None and no `AbstractStaticPartitionAssignor` is found.